images/scripts/services.setup

```diff
@@ -66,37 +66,37 @@ poweroff
 #
 #############
 
-cat <<EOF > /root/samba-ad.json
-{
-  "samba-container-config": "v0",
-  "configs": {
-    "demo": {
-      "instance_features": ["addc"],
-      "domain_settings": "sink",
-      "instance_name": "f0"
-    }
-  },
-  "domain_settings": {
-    "sink": {
-      "realm": "COCKPIT.LAN",
-      "short_domain": "COCKPIT",
-      "admin_password": "foobarFoo123"
-    }
-  }
-}
-EOF
-
-# See https://github.com/samba-in-kubernetes/samba-container#ad-dc
-podman pull quay.io/samba.org/samba-ad-server
+# See https://github.com/Fmstrat/samba-domain ; docker.io/nowsci/samba-domain works fine
+# but we have to build ourselves due to docker pull rate limits
+git clone --depth=1 https://github.com/Fmstrat/samba-domain /var/tmp/samba-domain
```

Comment:
Should we pin this at least?

Reply:
We didn't pin it in the past, and I'd actually like to get updates. We can pin it if/when it breaks, but I wouldn't like to do that in advance.

Reply:
Alright, approved! We pin it indirectly with a refresh so we are quite flexible in that regard.